BLSQ/iaso
# POLIO-672 No budget filter

Allow filtering on campaign with no budget (budget workflow not started)

    


## Self proof reading checklist

- [] Are my typescript files well typed
- [] Did I add translations
## Changes

We now represent the budget not being started by a - character instead of a null value in the database and in the API
which simplify the handling in the front and communication with the API
The migration will automatically convert existing campaign

your workflow need to be updated (see example workflow in the fixtures)

## How to test

See the set up instruction for budget workflow on the wiki
Try filtering campaign on budget page and doing a complete workflow.
